Userland/Libraries/LibELF/DynamicLinker.cpp

```diff
@@ -108,7 +110,22 @@ static Result<NonnullRefPtr<DynamicLoader>, DlErrorMessage> map_library(const St
         return map_library(name, fd);
     }
 
-    // TODO: Do we want to also look for libs in other paths too?
+    // Scan the LD_LIBRARY_PATH envvar if it exists or we're not a SUID program
+    if (s_ld_library_path != nullptr) {
```

Comment:
LD_LIBRARY_PATH should only be used if `AT_SECURE` was not specified.

Reply:
This is already the case. This value is set to `nullptr` and stays at that value unless `AT_SECURE` wasn't specified. See here: 

https://github.com/SerenityOS/serenity/blob/master/Userland/Libraries/LibELF/DynamicLinker.cpp#L508

Reply:
Huh, didn't notice that, that works then!